### PR TITLE
Python magics work on ints and lists

### PR DESCRIPTION
**Pull request by ioanaif**

### CODE REVIEW

File: src/xmagics/pythonexec.hpp [hunk 1]
```diff
@@ -27,14 +29,20 @@ namespace xcpp
...
+        void check_python_globals();
+        void exec_python_simple_command(const std::string code);
+        static std::string transfer_python_ints_utility();
+        static std::string transfer_python_lists_utility();
+        static bool python_check_for_initialisation();
 
     private:
 
-        void startup();
+        static void startup();
```
Comment (vgvassilev) [inline src/xmagics/pythonexec.hpp:43]: We probably need to make `update_python_dict_var` non static.

File: include/xeus-clang-repl/xinterpreter.hpp [hunk 2]
```diff
@@ -22,6 +22,7 @@
 #include "xeus_clang-repl_config.hpp"
 #include "xbuffer.hpp"
 #include "xmanager.hpp"
+#include "../../src/xmagics/pythonexec.hpp"
```
Comment (vgvassilev) [inline include/xeus-clang-repl/xinterpreter.hpp]: We probably want to move this file in `include`?

File: src/xmagics/pythonexec.hpp (hunk 1 above)
Comment (ioanaif) [reply src/xmagics/pythonexec.hpp:43]: It is used it `xinterpreter.cpp` that is why it is static.

Comment (vgvassilev) [reply src/xmagics/pythonexec.hpp:43]: Yes, it is used within a `static` function but I don't think the intent of this other function is to be non-static either.

**Review by vgvassilev — APPROVED** [review]
LGTM!

**Pull request merged**